src/test.rs

```diff
@@ -196,10 +204,32 @@ pub async fn krill_admin_expect_error(command: Command) -> Error {
     }
 }
 
-pub async fn refresh_all() {
+pub async fn cas_all_sync_parents() {
```

Comment:
Above there are separate sync and refresh functions. Here we have a sync function with a refresh command. Is this correct?